packages/common/src/BrowserSession.ts

```diff
@@ -133,6 +133,10 @@ export default abstract class BrowserSession extends BaseSession {
     })
   }
 
+  assureDeviceId(id: string, label: string, kind: MediaDeviceInfo['kind']): Promise<string> {
```

Comment:
@bryanrite is `assureDeviceId()` a good name? 

It could also be `validateDeviceId` or `checkDeviceId`..

Reply:
Reading the code, I think I slightly prefer `validateDeviceId`... assure _kind of_ implies to me that it will wait until the device is ready, rather than just checking to see if the device exists... but not enough to worry about much.

Reply:
I also prefer `validateDeviceId` so i'm going to change it 👍